### ISSUE TITLE
Make testing great again

### ISSUE THREAD

**Issue by igeligel** [gothinkster/vue-realworld-example-app#16]
Since the whole testing is not done, we should somehow make a plan to improve this.

Actually we should think of what to include, what to test, which types of test to include and so on. I am actually interested in e2e testing and capable of writing unit tests with `vue-test-utils` already. I guess e2e tests would be somehow similar.

Just want to hear some opinions on it 👍

**Comment by disjfa** [comment]
I'm not great at testing so any help and mentions are helful!

**Comment by igeligel** [comment]
Yeah I'll make some simple examples in the near future. Actually should I create issues for every component which should be tested and what should be tested? Then we can divide the tasks somehow and I think this would be a great way to contribute :)
